packages/breadcrumb/README.md

```diff
@@ -1,3 +1,5 @@
-# Breadcrumb Name
+# Breadcrumb
 
-Breadcrumb documentation
+Breadcrumbs are used for navigating within sections of an app. They communicate information hierarchy and help users follow the path they took to get to their current screen, and provide a way to navigate back to parent screens.
```

Comment:
nitpick: `They communicate [...] and help users [...], and provide a way [...]`. Something is off with the punctuation.